### DISCUSSION TITLE
Run PDM commands in a venv

### DISCUSSION THREAD

**Discussion by apollo13** [pdm-project/pdm#1494] [Q&A] [answered]
Hi there,

PDM let's one manage venvs which is nice. I know that I can activate the venvs manually but is there any option to the `pdm` cli so I can execute `run` or `sync` in a specified venv? Would it make sense to add something like this?

Thanks,
Florian

**Comment by frostming** [comment, marked as answer]
You probably won't be working with multiple venvs *at the same time* but one at a time. I don't see why `pdm use` won't work for you

**Reply by apollo13** [reply]
Ah perfect, thank you. I was looking for something like `pdm venv use`. It never occurred to me that I could use `pdm use` (though I now realize that it is mentioned in the note at https://pdm.fming.dev/latest/usage/venv/#activate-a-virtualenv ).
